mirroroperator/operator.py

```diff
@@ -62,10 +62,10 @@ def watch_registry_mirrors(self):
         hostess_docker_tag=os.environ.get("HOSTESS_DOCKER_TAG", "1.1.0"),
         # optional in V1PodSpec secrets split with comma
         image_pull_secrets=os.environ.get("IMAGE_PULL_SECRETS"),
-        # get secret name:
-        secret_name=os.environ.get("SECRET_NAME"),
-        # cert_name - needed in clusters
-        cert_name=os.environ.get("CERT_NAME"),
+        # get the docker certificate:
+        docker_certificate_secret=os.environ.get("DOCKER_CERTIFICATE_SECRET"),
```

Comment:
sorry to be awkward, but to me 'docker certificate secret' isn't clear. what does this mean?

I don't see an update to any documentation and docker uses secrets for many uses. Should this be registry_certificate_secret?

Reply:
Seems reasonable. :+1:

Reply:
We've just discussed this and don't really know what is best. One thing that did seem to be in favour of docker_certificate_secret was putting docker certificate into google gets you to the page which describes what we are creating here whereas registry certificate doesn't. I think the most important point was to update the docs, but we wanted to get the code finished before writing the docs for it.

Reply:
ok